content/en/user-guide/cloud-sandbox/ephemeral-instance/index.md

```diff
@@ -7,43 +7,36 @@ description: Create an Ephemeral Instance in the cloud using the LocalStack Web
 
 ## Introduction
 
-LocalStack Ephemeral Instance allows you to run an LocalStack instance in the cloud. You can interact with these remote instances via the LocalStack Web Application, or by configuring your integrations and developer tools with the endpoint URL of the remote instance.
-
-{{< callout >}}
-Ephemeral Instance is available on invite-only preview.
-If you'd like to try it out, please [contact us](https://www.localstack.cloud/demo) to request access.
-{{< /callout >}}
+Ephemeral Instance allows you to run an LocalStack instance in the cloud. You can interact with these remote instances via the LocalStack Web Application, or by configuring your integrations and developer tools with the endpoint URL of the remote instance.
 
 ## Getting started
 
 This guide is designed for users new to Ephemeral Instance and assumes basic knowledge of the LocalStack Web Application. In this guide, we will create an Ephemeral Instance and interact with it via the LocalStack Web Application and the AWS CLI.
 
 ### Create a new Ephemeral Instance
 
-Navigate to the [**LocalStack Instance Management**](https://app.localstack.cloud/instances) and click on the **Ephemeral (Team)** tab. Click on the **Create Ephemeral Instance** button.
+Navigate to the [**LocalStack Instance Management**](https://app.localstack.cloud/instances) and click on the **Ephemeral** tab. Enter the name of the new Ephemeral Instance, select the lifetime of the instance by dragging the slider, and click on **Launch**.
 
 <img src="ephemeral-instance-creation.png" alt="Creating an Ephemeral Instance" title="Creating an Ephemeral Instance" width="800" />
 
-You will be able to specify the name of the new Epheemeral Instance. Click on **Create Instance** to create the new Ephemeral Instance.
+Optionally, you can specify a LocalStack Extension to be installed in the Ephemeral Instance. You can select the extension from the **Extension settings** dropdown list before launching the Ephemeral Instance.
```

Comment:
```suggestion
Optionally, you can specify a LocalStack Extension to be installed in the Ephemeral Instance. You can select the extension from the **Extension settings** dropdown list before launching the Ephemeral Instance.
In case you have access to cloud pods and a pod you want to start your instance with, you can also choose it from the **Cloud Pod Settings** dropdown.
```

Reply:
there was a bug which didn't show the cloud pod dropdown